src/drivers/main.cpp

```diff
@@ -13,6 +13,7 @@ void printUsage()
   std::cerr << "Print Markdown reference :  binprecice md\n";
   std::cerr << "Print preCICE version    :  binprecice version\n";
   std::cerr << "                            binprecice --version\n";
+  std::cerr << "Check configuration file :  binprecice check FILE [ PARTICIPANT [ COMMSIZE ] ]\n";
```

Comment:
I am not sure if this is intended, but the style for `FILE [ PARTICIPANT [ COMMSIZE ] ]` is a bit different in the manpage.

Reply:
I don't mind any, I mainly pointed at the inconsistency.

Reply:
Let's leave it then as it is. Thanks for pointing it out though.